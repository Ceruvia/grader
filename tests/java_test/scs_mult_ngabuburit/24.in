4 2 5 1 3 1 1 3 3 4 2 0 2 