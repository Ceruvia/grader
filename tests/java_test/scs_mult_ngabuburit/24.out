Pintu dibuka!
Pergi ke luar rumah!
Laptop dinyalakan!
Menggunakan laptop untuk ngoding gacor
Menggunakan laptop untuk ngoding gacor
Pergi ke luar rumah!
Berlari di gasibu sejauh 10 km!
Membeli kolak!
Sudah jenuh! Lakukan aktivitas lain
